doc/making-a-visualizer.md

```diff
@@ -210,7 +252,9 @@ will show it in an error dialog.
 
 ### How to document your visualizer
 
-The p5 Template visualizer and the Differences visualizer both follow our
+Documentation should be included in the source file between markdown tags.
+This will be scraped from the source and automatically compiled into the
+online documentation. The p5 Template visualizer demonstrates our
 documentation conventions.
 
```

Comment:
Do you want to add a remark that it's the `/* md` comments that provide that demonstration, and that it also has detailed explanatory comments set off by `// ===` comments that are for the sake of providing more detail about visualizer writing, but are not needed in your own visualizers?

Reply:
good point